src/main/java/core/basesyntax/FigureDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDraw {

```

Comment:
```suggestion
public interface Drawable {
```
we will be able to draw not only figures in future